01week/javascripting/datatypes.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+/**
+Write a JavaScript program to display the current day and time.
+Write a JavaScript program to convert a number to a string.
+Write a JavaScript program to convert a string to the number.
+Write a JavaScript program that takes in different datatypes and prints out whether they are a:
+Boolean
+Null
+Undefined
+Number
+NaN
+String
+Write a JavaScript program that adds 2 numbers together.
+Write a JavaScript program that runs only when 2 things are true.
+Write a JavaScript program that runs when 1 of 2 things are true.
+Write a JavaScript program that runs when both things are not true.
+**/
+
+function currentDate() {          // displays the current date and time
+  const date = new Date();
```

Comment:
The const is redundant in this case, can be simplified into one line. `return new Date();`